code/game/objects/structures/fireaxe.dm

```diff
@@ -156,10 +156,23 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/fireaxecabinet, 32)
 	. = ..()
 	if(held_item)
 		. += item_overlay
+	var/hp_percent = atom_integrity/max_integrity * 100
```

Comment:
```suggestion
	var/hp_percent = (atom_integrity/max_integrity) * 100
```

lets fix this now because i remain eternally fearful of the pemdas demon